client/trino-client/src/test/java/io/trino/client/uri/TestTrinoUri.java

```diff
@@ -199,6 +199,18 @@ public void testInvalidUrls()
                 "Provided connection properties are invalid:\n" +
                         "Connection property assumeLiteralNamesInMetadataCallsForNonConformingClients cannot be set if assumeLiteralUnderscoreInMetadataCallsForNonConformingClients is enabled\n" +
                         "Connection property assumeLiteralUnderscoreInMetadataCallsForNonConformingClients cannot be set if assumeLiteralNamesInMetadataCallsForNonConformingClients is enabled");
+
+        // invalid validateConnection
```

Comment:
I don't think that all these negative tests are necessary. Just leave one string

Reply:
These tests are good for testing that we don't support boolean flags other than `true`/`false`. Not specific to `validateConnection`.

I just removed all test cases here.